lib/xboxrt/debug.h

```diff
@@ -19,6 +19,9 @@ extern "C"
 #define WHITE_16BPP   0xFFFF
 #define BLACK_16BPP  0x0000
 
+#define WHITE_15BPP   0xFFFF
```

Comment:
*Note: 0x7FFF would be enough, but setting alpha bit might be useful if `debugPrint` buffer is used as A1R5G5B5 texure in the future.*

Reply:
Using 0x7FFF (still untested), to match the 32bpp variant.